test/functional-tests/videostore.test.js

```diff
@@ -1,264 +1,264 @@
-/* eslint-disable */
```

Comment:
I suspect that _maybe_ this file is causing random errors. Commenting it out in case.